docs/Windows.md

```diff
@@ -38,7 +38,115 @@ Additionally, the ICU headers and libraries need to be provided for the build.
 ```
 
 ## MSVC
-To be filled in.
+- Windows doesn't currently have a build script. You'll need to run commands manually to build Swift on Windows.
+- Release/RelWithDebInfo modes have not been tested and may not be supported.
+- x86 has not been tested.
```

Comment:
Awesome! I meant a from-windows build (not X compilation) 

I know I'm having some problems with release mode not working, though - I hit an assertition compiling the stdlib